### ISSUE TITLE
[Bug]: pdfjs-dist not working in chrome

### ISSUE THREAD

**Issue by lhilgert9** [mozilla/pdf.js#18957]
### Attach (recommended) or Link to PDF file

Every PDF

### Web browser and its version

Google Chrome: 130.0.6723.59

### Operating system and its version

macOS 15.0.1

### PDF.js version

pdfjs-dist: 4.7.76

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

No

### Steps to reproduce the problem

1. use pdfjs-dist in a web-app as follows:
```ts
import * as pdfJS from 'pdfjs-dist';
pdfJS.GlobalWorkerOptions.workerSrc = window.location.origin + '/pdf.worker.min.js';

export default pdfJS;
```
2. build your own pdf viewer
3. open the web-app in chrome

### What is the expected behavior?

The file should be displayed correctly, like in safari.

### What went wrong?

```js
Uncaught TypeError: Failed to construct 'Headers': Invalid name
    at PDFNetworkStreamFullRequestReader._onHeadersReceived (pdfjs-dist.js?v=b25d4dee:10221:29)
    at NetworkManager.onStateChange (pdfjs-dist.js?v=b25d4dee:10109:22)
```

### Additional context

The error has only occurred since version 4.7.76. In version 4.6.82 everything works without errors.
Looks like the error occurred [here](https://github.com/mozilla/pdf.js/blob/master/src/display/network.js#L276-L285).

**Comment by nicolo-ribaudo** [comment]
https://mozilla.github.io/pdf.js/web/viewer.html works in Chrome without any problem. Please provide a link to a reproduction.

**Comment by timvandermeij** [comment]
Closing until more information is provided, as requested in the issue template. Without it this issue is unfortunately not actionable for us.

**Comment by CyberAndrii** [comment]
Made a PR that should fix this: https://github.com/mozilla/pdf.js/pull/19010

**Comment by juliasleeps** [comment]
> Made a PR that should fix this: [#19010](https://github.com/mozilla/pdf.js/pull/19010)

hello
seems additional fix is needed

const [key, ...val] = x.split(": "); (https://github.com/mozilla/pdf.js/commit/840cc5e0d42dfc97ea0317ca3cf37dff586acd39#diff-73a7d133cef0fa46b7465f7fa8eb81633d03d7ff1fc2035b0e0437ffb987cacaR282)

splitting headers with having space after colon **": "**  producing exception if fullRequestXhr.getAllResponseHeaders() returns headers string like:
"content-length: 84385\r\ncontent-type: \r\n"
in this case new Header are creating as new Headers([["content-length", "84385"], ["content-type:", ""]]);
causing Uncaught TypeError: Failed to construct '**Headers': Invalid name**"
